app/views/doctors/show.html.erb

```diff
@@ -0,0 +1,15 @@
+<h1>Doctor: <%= @doctor.name %></h1>
+
+<p>Specialty: <%= @doctor.specialty %></p>
+<p>Graduate of: <%= @doctor.university %></p>
+
+<p>Doctor at: <%= @doctor.hospital.name %></p>
+
+<% @doctor.patients.each do |patient| %>
+<section id="patient-<%=patient.id%>">
+  <li>
+    <%= patient.name %>
+    <%= button_to "Remove #{patient.name}", "/doctors/#{@doctor.id}/patients/#{patient.id}", params: {doctor_id: @doctor.id, patient_id: patient.id}, method: :delete %>
```

Comment:
With these IDs in the path, you don't need to send the parameters at all! Just fyi